specification/network/resource-manager/Microsoft.Network/stable/2022-07-01/azureFirewall.json

```diff
@@ -437,6 +437,66 @@
           "final-state-via": "location"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/azureFirewalls/{azureFirewallName}/packetCapture": {
+      "post": {
+        "tags": [
+          "AzureFirewalls"
+        ],
+        "operationId": "AzureFirewalls_PacketCapture",
+        "description": "Runs a packet capture on AzureFirewall.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "azureFirewallName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Azure Firewall."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/FirewallPacketCaptureParameters"
+            },
+            "description": "Parameters supplied to update azure firewall tags."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted and the operation will complete asynchronously."
+          },
```

Comment:
need to also include the 200 for the final status returned by the location header

Reply:
Just bring this comment back here from the older PR : https://github.com/Azure/azure-rest-api-specs/pull/20601

This 200 refers to the final state that the client will get when it does the GET on the location header that you return as part of the 202

Reply:
I believe i addressed the changes you asked for
please take a look and request an example if something is incorrect. Thanks!

Reply:
added 200 please let me know if this is what was needed. Or is something else needed?